### PR TITLE
#169258569 update circleCI

### PR DESCRIPTION
**Pull request by victkarangwa**
#### What does this PR do?
The PR integrate CircleCI with the project

#### Description of Task to be completed?
  - Add CircleCI badge on README
  - Add mock test 

#### How should this be manually tested?
In order to test the application try the following steps
- Run `git clone https://github.com/andela/team-odd-bn-frontend/tree/ch-setup-circleCI-169258569.git`
- `cd team-odd-bn-frontend`
- `ch-setup-circleCI-169258569`
- `npm install`
- Lastly, run `npm run dev` 

#### What are the relevant pivotal tracker stories?
[#169258569](https://www.pivotaltracker.com/story/show/169258569)
#### Screenshots (if appropriate)
N/A
#### Questions:

### CODE REVIEW

**Review by NiyongaboEric — APPROVED** [review]
@victorkarangwa4 I am happy you configured CI. I think you are good to go!

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,12 @@
+<<<<<<< HEAD
```
Comment (hezronkimutai) [inline .gitignore]: @victorkarangwa4 , can you resolve this conflict.

**Review by hezronkimutai — CHANGES_REQUESTED** [review]
Nice work, just resolve the conflict in .gitignore

**Review by hezronkimutai — APPROVED** [review]

**Review by nziokaivy — APPROVED** [review]
Looks good to me @victorkarangwa4!

**Review by shemaeric — APPROVED** [review]
Good work @victorkarangwa4

**Pull request merged**